documentation/Functions.md

```diff
@@ -194,9 +194,11 @@ buildJava("cobigenexample", true)
 ***
 
 ### createFile <a name="createFile"></a>
+This function create a file structure at workspace directory.
```

Comment:
I would prefer something like:
`This function creates a new file with a specified content. If no content is specified, an empty file is created.`